frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayEnumBox.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Siemens AG, 2017.
+ * Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.portal.tags;
+
+import org.apache.thrift.TEnum;
+import org.eclipse.sw360.datahandler.common.ThriftEnumUtils;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.tagext.SimpleTagSupport;
+import java.io.IOException;
+
+public class DisplayEnumBox extends SimpleTagSupport {
+
+    private TEnum value;
+
+    public void setValue(TEnum value) {
+        this.value = value;
+    }
+
+    public String makeBox(String state) {
+        String projectStateBackgroundColour = "backgroundGrey";
+        if( state == "Active" ) {
```

Comment:
If you are going to change the type, I think a direct comparison of `value == ProjectState.ACTIVE` would be more reliable than string comparison here. Otherwise, this code will break if someone ever changes the string representation of `ProjectState.ACTIVE` in `ThriftEnumUtils`.